deployment/CP04OSPM_Deploy.csv

```diff
@@ -91,21 +91,21 @@ AR34,,AR39,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:0
-AR44,,,,CP04OSPM-WFP01-05-PARADK000,13,1,2020-06-07T00:00:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,,,,,,
+AR39,,AR44,,CP04OSPM-RII01-02-ADCPSL010,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,,CGINS-ADCPSL-21393,39.935558,-70.880328,,406,451,
+AR39,,AR44,,CP04OSPM-SBS01-00-STCENG000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,,CGCON-BSTC01-00012,39.935558,-70.880328,,0,451,
+AR39,,AR44,,CP04OSPM-SBS11-02-MOPAK0000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,,CGINS-MOPAK0-11347,39.935558,-70.880328,,0,451,
+AR39,,AR44,,CP04OSPM-WFP01-00-WFPENG000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.935558,-70.880328,,405,451,
+AR39,,AR44,,CP04OSPM-WFP01-01-VEL3DK000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.935558,-70.880328,,405,451,
+AR39,,AR44,,CP04OSPM-WFP01-02-DOFSTK000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.935558,-70.880328,,405,451,
+AR39,,AR44,,CP04OSPM-WFP01-03-CTDPFK000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.935558,-70.880328,,405,451,
+AR39,,AR44,,CP04OSPM-WFP01-04-FLORTK000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.935558,-70.880328,,405,451,
+AR39,,AR44,,CP04OSPM-WFP01-05-PARADK000,12,1,2019-10-13T13:42:00,2020-06-08T16:19:00,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-PARADK-20513,39.935558,-70.880328,,405,451,
+AR44,,,,CP04OSPM-RII01-02-ADCPSL010,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,,CGINS-ADCPSL-21446,39.943117,-70.88605,,418,438,
+AR44,,,,CP04OSPM-SBS01-00-STCENG000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,,CGCON-BSTC01-00004,39.943117,-70.88605,,0,438,
+AR44,,,,CP04OSPM-SBS11-02-MOPAK0000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,,CGINS-MOPAK0-11357,39.943117,-70.88605,,0,438,
+AR44,,,,CP04OSPM-WFP01-00-WFPENG000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-01-VEL3DK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-02-DOFSTK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-03-CTDPFK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-04-FLORTK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.943117,-70.88605,,417,438,
+AR44,,,,CP04OSPM-WFP01-05-PARADK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.943117,-70.88605,,417,438,
```

Comment:
looks good.  Are the sensor depths correct for previous deployment (OSPM-12)?

Reply:
For the deep site (nominally 460 m), the ADCP should be 45 m up from the seafloor.  So for a a 451 m deployment depth, the ADCP should be at 406, which it is.